shop-info-service/src/functions/getProductById/index.ts

```diff
@@ -0,0 +1,14 @@
+import { handlerPath } from '@libs/handler-resolver';
+
+export const getProductById = {
+  handler: `${handlerPath(__dirname)}/handler.getProductById`,
+  events: [
+    {
+      http: {
+        method: 'get',
+        path: 'get-product-by-id',
```

Comment:
The requested URL should be /products/{productId}

Reply:
I noted this point in the README, it's made on purpose

Reply:
updated